File: Java/Program/Clothing.java
public class Clothing extends Product {
    protected String size;      // Ukuran pakaian
    protected String material;  // Bahan pakaian
    protected String gender;    // Jenis kelamin pakaian

    Clothing(){  // Konstruktor kosong
        
    }
    
    // Konstruktor
    public Clothing(String idProduct, String name, String brand, double price, String size, String material, String gender) {
        super(idProduct, name, brand, price);  // Memanggil konstruktor kelas induk dengan parameter yang sesuai
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // Setter untuk ukuran pakaian
    public void setSize(String size) {
        this.size = size;
    }

    // Setter untuk bahan pakaian
    public void setMaterial(String material) {
        this.material = material;
    }

    // Setter untuk jenis kelamin pakaian
    public void setGender(String gender) {
        this.gender = gender;
    }

    // Getter untuk ukuran pakaian
    public String getSize() {
        return size;
    }

    // Getter untuk bahan pakaian
    public String getMaterial() {
        return material;
    }

    // Getter untuk jenis kelamin pakaian
    public String getGender() {
        return gender;
    }
}

File: Python/Program/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self, idProduct="", name="", brand="", price=0.0, size="", material="", gender=""):
        super().__init__(idProduct, name, brand, price) #super untuk memanggil atribut di class product
        self.size = size                                #atribut untuk menampung size
        self.material = material                        #atribut untuk menampung material
        self.gender = gender                            #atribut untuk menampung gender

    #Getter dan Setter
    def setSize(self, size):
        self.size = size

    def setMaterial(self, material):
        self.material = material

    def setGender(self, gender):
        self.gender = gender

    def getSize(self):
        return self.size

    def getMaterial(self):
        return self.material

    def getGender(self):
        return self.gender
